platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ExportMigrationConfigurationAdminContext.java

```diff
@@ -0,0 +1,171 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.configuration.admin;
+
+import java.io.File;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Path;
+import java.nio.file.PathMatcher;
+import java.nio.file.Paths;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.apache.commons.io.FilenameUtils;
+import org.apache.commons.lang.Validate;
+import org.apache.felix.fileinstall.internal.DirectoryWatcher;
+import org.codice.ddf.configuration.persistence.PersistenceStrategy;
+import org.codice.ddf.migration.ExportMigrationContext;
+import org.codice.ddf.migration.ExportMigrationEntry;
+import org.codice.ddf.migration.MigrationEntry;
+import org.codice.ddf.migration.MigrationWarning;
+import org.codice.ddf.migration.ProxyExportMigrationContext;
+import org.osgi.service.cm.Configuration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ExportMigrationConfigurationAdminContext extends ProxyExportMigrationContext {
+    private static final Logger LOGGER = LoggerFactory.getLogger(
+            ExportMigrationConfigurationAdminContext.class);
+
+    private final ConfigurationAdminMigratable admin;
+
+    private final Set<String> warnedExtensions = new HashSet<>(12);
```

Comment:
It could be smaller. I just don't want to let the default constructor initialize with too many entries when I know it will never be too big to start with.

Changed it to 8